src/tests/ftest/launch.py

```diff
@@ -333,6 +333,14 @@ def get_available_interfaces(args):
         print("Error obtaining a default interface on {} from {}".format(all_hosts, net_path))
 
     print("Available interfaces on {}: {}".format(all_hosts, available_interfaces))
+
+    # pylint: disable=import-outside-toplevel
+    sys.path.append(os.path.join(os.getcwd(), "util"))
+    from network_utils import get_fastest_interface
+    print("<>" * 50)
+    print("get_fastest_interface: {}".format(get_fastest_interface(all_hosts)))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string